src/xmlutil.rs

```diff
@@ -250,4 +282,23 @@ mod tests {
         }
     }
 
+    #[test]
+    fn skip_tree_with_subtree() {
+        let file = File::open("tests/sample-data/skip_tree.xml").unwrap();
+        let file = BufReader::new(file);
+        let mut my_parser  = EventReader::new(file);
+        let my_stack = my_parser.events().peekable();
+        let mut reader = XmlResponseFromFile::new(my_stack);
+
+        // skip two leading fields since we ignore them (xml declaration, return type declaration)
+        reader.next();
+        reader.next();
```

Comment:
With only one `reader.next()`, the test fails with:

    ---- xmlutil::tests::skip_tree_with_subtree stdout ----
    thread 'xmlutil::tests::skip_tree_with_subtree' panicked at 'Missing tag f: XmlParseError("Expected StartElement f, got Some(EndDocument)")', src/libcore/result.rs:746

I added some `println!()`s to see the values returned in every `next()`:

    Some(StartDocument(1.0, UTF-8, None))
    Some(StartElement(a, {Some("xml"): "http://www.w3.org/XML/1998/namespace", Some("xmlns"): "http://www.w3.org/2000/xmlns/", None: ""}))

So, the next event will be `StartElement(<b>)`, which is the tree that I want to skip.

In this implementation, `skip_tree` has to be called when the next event is the element that open the tree to ignored. I don't know if this is the best way to implement it, but at least it is the easiest one that I could find; I'm open to suggestions.